### PR TITLE
Added US_DRIVER_LICENSE test case

### LINKED ISSUE #19: Detect US_DRIVER_LICENSE
A U.S. driver’s license is a type of personal information that allows someone to legally drive a car. Issued by the state's Department of Motor Vehicles (DMV), it shows important details like the person’s name, photo, address, birth date, and a special ID number. This license is also used for other things, like proving who you are when you open a bank account, apply for a job, get on a plane, or buy certain products. Because it has so much personal information, it needs to be kept safe to avoid identity theft and fraud.

see [here](https://www.state.gov/driver-services/) for more information about US_DRIVER_LICENSE .

### PR DESCRIPTION
**Pull request by brendaar1991**

### CODE REVIEW

**Comment by brendaar1991** [comment]
Deleted the old branch that was created on test_null and created this one that was created on the right test_traveling_sales.py

File: test_traveling_sales.py [hunk 1]
```diff
@@ -17,49 +17,35 @@ def test_anonymize_nrp(self):
         self.assertEqual('My NRP is Unknown',
                          anonymize_text('My NRP is Unknown', ['NRP']))
         
-    def test_au_medicare(self):
```
Comment (LailaRH20) [inline test_traveling_sales.py:20]: original code. not sure if this is necessarily something you should be concerned with.

**Review by LailaRH20 — COMMENTED** [review]
great!

**Review by mljones5 — APPROVED** [review]

**Review by LailaRH20 — APPROVED** [review]
I did not press the correct button last time

**Pull request merged**